cmd/supervisor/supervisor.go

```diff
@@ -0,0 +1,16 @@
+// (c) Cartesi and individual authors (see AUTHORS)
+// SPDX-License-Identifier: Apache-2.0 (see LICENSE)
+
+package main
+
+import (
+	"os"
+
+	cmd "github.com/cartesi/rollups-node/internal/pkg/supervisor"
+)
+
+func main() {
```

Comment:
It would be clearer if the file containing the main function is called `main.go`.